nvim-lspconfig-scm-1.rockspec

```diff
@@ -20,16 +20,9 @@ dependencies = {
 }
 
 source = {
-  url = 'https://github.com/neovim/nvim-lspconfig/archive/v' .. _MODREV .. '.zip',
-  dir = 'nvim-lspconfig-' .. _MODREV,
+  url = 'git://github.com/neovim/nvim-lspconfig',
```

Comment:
have you tested this pattern work with tagged release too ?

Reply:
The luarocks-tag-release action doesn't use the `scm-1.rockspec` so there's no need to handle sources for tagged releases in this file.